chat-model/src/main/resources/canon/chat.json

```diff
@@ -1261,6 +1261,29 @@
           }
         }
       },
+      "ReactionDetailsPayload": {
+        "type": "object",
+        "extends": "#/components/schemas/MaestroPayload",
+        "properties": {
+          "action": {
+            "type": "string"
```

Comment:
I would define an enum for this although I won't object if you don't want to.

Reply:
We would prefer not to define an enum for now, as the accepted values will be extended in the future release.